ui/admin/app/routes/scopes/scope/targets.js

```diff
@@ -51,16 +51,30 @@ export default class ScopesScopeTargetsRoute extends Route {
   /**
    * Handle save.
    * @param {TargetModel} target
-   * @param {Event} e
+   * @param {TargetModel} egressEnabled
    */
   @action
   @loading
   @notifyError(({ message }) => message)
   @notifySuccess(({ isNew }) =>
     isNew ? 'notifications.create-success' : 'notifications.save-success'
   )
-  async save(target) {
-    await target.save();
+  async save(target, egressEnabled, updateDeprecatedWorkerFilter) {
```

Comment:
The existing route save action should remain the same as always.  One way to achieve this is if the form component accepts an additional argument and switches the save action that it calls.